packages/components/src/components/post-collapsible/post-collapsible.tsx

```diff
@@ -40,7 +40,15 @@ export class PostCollapsible {
       'boolean',
       'The `collapsed` property of the `post-collapsible` must be a boolean.',
     );
-
+    
+    if (!this.isLoaded) {
+      Object.assign(
+        this.host.style, 
+        this.collapsed ? collapsedKeyframe : expandedKeyframe
+      );
+      this.isOpen = !this.collapsed;
+    }
```

Comment:
A `return` or `else` statement is missing here as you do not want the `toggle` method to be called when the collapsible is not loaded.